package.json

```diff
@@ -43,13 +43,13 @@
     "@typescript-eslint/eslint-plugin": "4.x",
     "@typescript-eslint/parser": "4.x",
     "babel-eslint": "10.x",
-    "eslint": "7.x",
-    "eslint-plugin-babel": "5.x",
-    "eslint-plugin-enact": "0.x",
-    "eslint-plugin-jsx-a11y": "6.3.x",
-    "eslint-plugin-react": "7.x",
-    "eslint-plugin-react-hooks": "4.x",
+    "eslint": "^7.32.0",
```

Comment:
Please update peerDependencies too.

Reply:
I think there's no need to change peerDependencies